docker/python3/focal/Dockerfile

```diff
@@ -79,6 +79,19 @@ RUN apt-get update \
     && pip3 install --user six==1.14.0 \
     && rm -rf /var/lib/apt/lists/* \
     && apt-get clean -y
+
+ENV NODE_VERSION=16.13.0
```

Comment:
Latest stable LTS version of node is 18.18.1.   It would be nice to upgrade, unless there are site-docs dependencies or other repository dependencies.  Not critical.